diff --git a/node_modules/ffmpeg-kit-react-native/android/build.gradle b/node_modules/ffmpeg-kit-react-native/android/build.gradle
index 2909280..23a1366 100644
--- a/node_modules/ffmpeg-kit-react-native/android/build.gradle
+++ b/node_modules/ffmpeg-kit-react-native/android/build.gradle
@@ -125,5 +125,6 @@ repositories {
 
 dependencies {
   api 'com.facebook.react:react-native:+'
-  implementation 'com.arthenica:ffmpeg-kit-' + safePackageName(safeExtGet('ffmpegKitPackage', 'https')) + ':' + safePackageVersion(safeExtGet('ffmpegKitPackage', 'https'))
+  implementation 'com.moizhassan.ffmpeg:ffmpeg-kit-16kb:6.1.1'
 }
+
